fusor-ember-cli/app/serializers/entitlement.js

```diff
@@ -1,33 +1,25 @@
 import DS from 'ember-data';
 
-export default DS.RESTSerializer.extend({
-
-  // add root node 'entitlements' that customer protal JSON response doesn't return
-  extractArray(store, type, payload) {
-    payload = { entitlements: payload };
-    return this._super(store, type, payload);
-  },
+export default DS.JSONSerializer.extend({
```

Comment:
`JSONSerializer` is a new serializer that does NOT expect a root node, so we don't need to manipulate the result to add a node anymore.